packages/@guidesmiths/cuckoojs-schematics/src/systemic-scaffolding/files/opentelemetry.js

```diff
@@ -0,0 +1,20 @@
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
+const { PinoInstrumentation } = require('@opentelemetry/instrumentation-pino');
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+
+const init = () => {
+  const provider = new NodeTracerProvider();
+  provider.register();
```

Comment:
In the same link [above](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node#readme) it's shown how to add a service name for telemetry data
`
const { Resource } = require('@opentelemetry/resources');
const provider = new NodeTracerProvider({
  resource: new Resource({
    [SemanticResourceAttributes.SERVICE_NAME]: 'basic-service',
  }),
});`

Reply:
I removed that part intentionally because the initial approach is to use instrumentation limited to the app's scope. I think it wouldn't hurt to add it by default 👍